src/node.cc

```diff
@@ -4270,7 +4263,19 @@ static void StartNodeInstance(void* arg) {
                              array_buffer_allocator.zero_fill_field());
     Local<Context> context = Context::New(isolate);
     Context::Scope context_scope(context);
+
     Environment env(&isolate_data, context);
+
+    uv_async_init(env.event_loop(),
+                  env.dispatch_debug_messages_async(),
+                  DispatchDebugMessagesAsyncCallback);
+    uv_unref(reinterpret_cast<uv_handle_t*>(
+             env.dispatch_debug_messages_async()));
+
+    if (!use_debug_agent) {
```

Comment:
You might as well use `instance_data->use_debug_agent()` now that it's been moved to StartNodeInstance().